### DISCUSSION TITLE
Support for Disfluencies (Filler Words) in Text-to-Speech Output

### DISCUSSION THREAD

**Discussion by nitin-airindia** [deepgram/community#954] [General help] [answered]
Currently, the Deepgram Text-to-Speech API sometimes ignores or spells out disfluencies like 'oh', 'uh', and 'um', when I try using demo available of deepgram.com. I want to include these filler words in the spoken output, making the speech sound more natural and conversational. Is it possible?

**Comment by deepgram-community** [comment, marked as answer]
hey there. using lowercase filler words should consistently be turned into audio for the filler words. If you give it uppercase UM or UH then it will spell them out. think of the input text as text to prompt the service rather than text a user would read.

You can check out our prompting guide here: <https://developers.deepgram.com/docs/text-to-speech-prompting>

> luke
